velox/tool/trace/PartitionedOutputReplayer.cpp

```diff
@@ -150,6 +150,7 @@ RowVectorPtr PartitionedOutputReplayer::run() {
       executor_.get(),
       consumerExecutor_.get(),
       consumerCb_);
+  printStats(task);
```

Comment:
This should be a common thing. Should we abstract it out to put it in the framework in some way? Does not seem to be clean to make individual calls in every single impl. Newly added impls might miss this printing.